### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.8 es 4.43 (6 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#316] [Esercizi M-Z]
![es.43](https://github.com/user-attachments/assets/a4a051e3-476b-427b-8ca8-2fc8ce54fc3e)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/ec2e736a-f8a0-4829-96cc-ae954dcef195)

**Reply by alemilos** [reply]
Forse una volta trovato che per ogni gruppo di 5 bit c'è una probabilità di 0,05792 che sia errato, bisogna calcolare per n gruppi di 5 bit, quindi un messaggio vero e proprio.
1 - (1-0,05792)^n è la probabilità che il messaggio sia errato, ovvero abbia almeno un gruppo di 5 bit che viene decodificato erronamente. Ha senso ?
